test/Test/S.purs

```diff
@@ -63,7 +64,9 @@ searchTest = do
             {include: true, labels: [Common "foo"],
              predicate: Contains (Text "uni*")},
             {include: true, labels: [Meta "path"],
-             predicate: Contains (Text "/foo/bar")}
+             predicate: Contains (Text "/foo/bar")},
+            {include: true, labels: [Common "path"],
+             predicate: Contains (Text "\"foo bar\"")}
```

Comment:
Good point. These quotes are purely for the parser to allow special characters, so they shouldn't appear in the ADT. @cryogenian Can you do a separate PR to fix this?